helm/net-exporter/templates/psp.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: policy/v1beta1
```

Comment:
Available since 1.10 https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/